src/__test__/index.test.ts

```diff
@@ -31,6 +31,37 @@ describe('graphqlify', () => {
     `)
   })
 
+
+  it('render GraphQL alias', () => {
+    const queryObject = {
+      [alias('maleUser', 'user')]: {
+        __params: { id: 1 },
+        id: types.number,
+        name: types.string,
+        bankAccount: {
+          id: types.number,
+          branch: types.string,
+        },
+      }
+    }
+    console.log('queryObject', queryObject)
```

Comment:
Could you remove console.log?

Reply:
oh! I realized that there is no lint config just now! I used to lint TS by eslint TS rule. Which style do you like?